airflow-core/src/airflow/ui/src/i18n/locales/zh-TW/dag.json

```
@@ -0,0 +1,110 @@
+{
+  "allRuns": "所有運行",
```

```suggestion
  "allRuns": "所有執行",
```

I think we use "執行" elsewhere. we should unify them